third_party/ibis/ibis_addon/operations.py

```diff
@@ -66,6 +66,10 @@ class HashBytes(ValueOp):
     how = Arg(rlz.isin({"sha256", "farm_fingerprint"}))
     output_type = rlz.shape_like("arg", "binary")
 
+class to_char(ValueOp):
+    arg = Arg(rlz.one_of([rlz.value(dt.Decimal), rlz.value(dt.float64)]))
+    fmt = Arg(rlz.isin({"FM90.099"}))
```

Comment:
I am unable to validate for the fmt to be string. Tried many rules but nothing seem to work for me. So I kept it to rlz.isin as of now. Might need some help for other validations, if required.